infrastructure/main.tf

```diff
@@ -0,0 +1,67 @@
+terraform {
+  required_version = "~> 1.1.2"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.90"
+    }
+  }
+
+  backend "azurerm" {
+    resource_group_name  = "ioet-infra-tf-state"
+    storage_account_name = "timetrackertfstate"
+    container_name       = "time-tracker-tf-state"
+    key                  = "time-tracker-ui.tfstate"
+  }
+
+}
+
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+}
+
+data "terraform_remote_state" "service" {
+  backend   = "azurerm"
+  workspace = terraform.workspace
+  config = {
+    resource_group_name  = "ioet-infra-tf-state"
+    storage_account_name = "timetrackertfstate"
+    container_name       = "time-tracker-tf-state"
+    key                  = "this.tfstate"
+  }
+}
+
+data "azurerm_container_registry" "registry" {
```

Comment:
**issue**: I think this will make another unnecessary call to the azure API. The `data.terraform_remote_state.service.outputs.container_registry_name` already provides a valid reference to the container registry

Reply:
really? i change that